cmd/packageExport.go

```diff
@@ -73,18 +73,18 @@ func newPackageExportCommand() *packageExportCommand {
 		},
 	}
 
-	cmd.Flags().BoolVarP(&template, "template", "t", false, "Export a package config template")
+	cmd.Flags().BoolVarP(&template, "example", "e", false, "Export a package config example")
```

Comment:
Varibale name `template` should be changed to `example` too. Thanks for your work!

Reply:
Sorry I missed some stuff there :/ Should look better now

Reply:
No worries, great job!